### PR TITLE
Readme corrections

### PR DESCRIPTION
**Pull request by XomaDev**
Improved the readme and corrected the typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,17 +94,17 @@ After you add the installation path to the PATH variable, open a new terminal (
    ```
    netstat -ano | findstr :8888
    ```
-   In your prefered terminal, you will get a simmilar output:
+   In your prefered terminal, you will get a similar output:
    ```
     TCP    0.0.0.0:8888           0.0.0.0:0              LISTENING       17692
     TCP    YOUR_LOCAL_IP:60121        127.0.0.1:8888         TIME_WAIT       0
-    TCP    YOUR_LOCAL_IP:63783        127.0.0.1:8888         TIME_WAIT       0
+    TCP    YOUR_LOCAL_IP:63783        127.0.0.1:88Windows88         TIME_WAIT       0
```
Comment (MohamedTamer94) [inline README.md]: The port `8888` should be correct, not sure why placing `Windows` here?

File: README.md [hunk 2]
```diff
@@ -73,19 +73,19 @@ After you add the installation path to the PATH variable, open a new terminal (
...
-  - `appzard doctor`: appzard doctor requires internet connection, to check the latest version of appzard.
+  The advantage of appzard offline is to be able to run appzard workspace locally without an internet connection. However, some other functions require an internet connection to work properly:
+  - Installation: installing Appzard requires an internet connection to download installation files from the cloud.
+  - Upgrade: upgrading appzard requires an internet connection, to check the latest version, and download the updated files.
+  - `appzard doctor`: appzard doctor requires an internet connection, to check the latest version of appzard.
   ### How to stop appzard offline instance
   When starting a new instance you might have got a similar error: `An appzard instance is already running!`
   
-  This error is typically resulted in from two situations:
+  This error typically results in two situations:
```
Comment (MohamedTamer94) [inline README.md]: Actually the error is resulted in from the two situations, not the situations are resulted in from the error, so the previous one should be correct.
```suggestion
  This error is typically resulted in from two situations:
```
Or?
```suggestion
  This error is typically caused by two situations:
```

**Review by MohamedTamer94 — CHANGES_REQUESTED** [review]
Thanks for this PR! It should be fine to merge after addressing the feedback below:
P.S:
for further corrections for typos, it's better to create an issue.

File: README.md (hunk 2 above)
Comment (XomaDev) [reply README.md]: okay, thanks

**Review by MohamedTamer94 — APPROVED** [review]
LGTM

**Pull request merged**